Commerce 1430 Add approval workflow to mini cart

alecsloan opened the pull request.

alecsloan committed:
COMMERCE-1430 Add approval workflow handling in Mini Cart and add 'Submit' button in place of 'Checkout'

alecsloan committed:
COMMERCE-1430 Semantic Versionsing (gradle baseline)

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

ethanbustad commented:
ci:test

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x-next">7.1.x-next</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-21.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/118/">test-subrepository-acceptance-pullrequest(7.1.x)</a></p><p><b>For more details click <a href="https://test-1-21.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/118/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h1>Validation FAILED.</h1><p>Build Time: 23 minutes 39 seconds 706 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/01be81a07d11c9ecc960a659c87993adf38914f1">01be81a07d11c9ecc960a659c87993adf38914f1</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :heavy_check_mark:</li><li>subrepository-pmd-jdk8 -  :heavy_check_mark:</li><li>subrepository-source-format-jdk8 -  :x:<pre><code>     [echo] java.lang.Exception: Found 3 formatting issues:
     [echo] 1: ./commerce-order-content-web/build.gradle expected:&lt;...s:commerce:commerce-[frontend-taglib")
     [echo] 	compileOnly project(":apps:commerce:commerce-product-api]")
     [echo] }&gt; but was:&lt;...s:commerce:commerce-[product-api")
     [echo] 	compileOnly project(":apps:commerce:commerce-frontend-taglib]")
     [echo] }&gt;
     [echo] 2: ./commerce-service/build.gradle expected:&lt;...s:commerce:commerce-[api")
     [echo] 	compileOnly project(":apps:commerce:commerce-product]-api")
     [echo] }&gt; but was:&lt;...s:commerce:commerce-[product-api")
     [echo] 	compileOnly project(":apps:commerce:commerce]-api")
     [echo] }&gt;
     [echo] 3: ./commerce-frontend-taglib/build.gradle expected:&lt;...s:commerce:commerce-[api")
     [echo] 	compileOnly project(":apps:commerce:commerce-product]-api")
     [echo] }&gt; but was:&lt;...s:commerce:commerce-[product-api")
     [echo] 	compileOnly project(":apps:commerce:commerce]-api")
     [echo] }&gt;
     [echo] 
     [echo] 	at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:438)
     [echo] 	at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:266)
     [echo] 
     [echo] FAILURE: Build failed with an exception.
     [echo] 
     [echo] * What went wrong:
     [echo] Execution failed for task ':checkSourceFormatting'.
     [echo] &gt; Process 'command '/opt/java/jdk1.8.0_121-x64/bin/java'' finished with non-zero exit value 1
     [echo] 
     [echo] * Try:
     [echo] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [echo] 
     [echo] * Exception is:
     [echo] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':checkSourceFormatting'.
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [echo] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)</code></pre></li><li>central-requirements-jdk8/subrepository-unit-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-13.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/72//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>10 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-13.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/72//testReport">here</a>.</h5></div></html>

alecsloan commented:
ci:test

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x-next">7.1.x-next</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-13.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/74/">test-subrepository-acceptance-pullrequest(7.1.x)</a></p><p><b>For more details click <a href="https://test-1-13.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/74/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h1>Validation PASSED. Running batch tests.</h1><p>Build Time: 27 minutes 35 seconds 123 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/01be81a07d11c9ecc960a659c87993adf38914f1">01be81a07d11c9ecc960a659c87993adf38914f1</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :heavy_check_mark:</li><li>subrepository-pmd-jdk8 -  :heavy_check_mark:</li><li>subrepository-source-format-jdk8 -  :heavy_check_mark:</li><li>central-requirements-jdk8/subrepository-unit-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-11.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/157//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>10 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-11.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/157//testReport">here</a>.</h5></div></html>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test - 6 out of 6 jobs passed in 1 hour 15 minutes 38 seconds 774 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/01be81a07d11c9ecc960a659c87993adf38914f1">01be81a07d11c9ecc960a659c87993adf38914f1</a></p>6 out of 6 jobs PASSED<details><summary><strong>6 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-13.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/74/">test-subrepository-acceptance-pullrequest(7.1.x)</a></li><li><a href="https://test-1-20.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/555/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/572/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-assemble-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/556/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-integration-mysql57-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-subrepository-acceptance-pullrequest-dist(7.1.x)/44/">test-subrepository-acceptance-pullrequest-dist(7.1.x)/jdk8</a></li><li><a href="https://test-1-11.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/157/">test-subrepository-acceptance-pullrequest-validation(7.1.x)/jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-13.liferay.com/userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/74/jenkins-report.html">here</a>.</h5></details></html>

brianchandotcom committed:
Commerce 2.0.1 bundles

alecsloan committed:
COMMERCE-1605 Fix the groupId that we're using in our searchContext

Commit:
COMMERCE-1608 commerce - headless - admin catalog - FIX BUG - properly stream to String array (to avoid cast exceptions in runtime)

Commit:
COMMERCE-1615 headless-commerce-catalog-admin - make sure nested optional objects are not null

brianchandotcom committed:
LPS-85973 Use a real version of vulcan/odata api

Commit:
COMMERCE-1594 Wrap persistence calls in transaction invoker

Commit:
COMMERCE-872 RELEASE-1645 Bump JS versions

Commit:
COMMERCE-1594 Mark as a temporary fix pending LPS-97955

We'll want to replace this with a local service call once the
service method is published and available to consumers.

hhuijser committed:
Merge branch '7.1.x-next' of github.com:liferay/com-liferay-commerce into 7.1.x-next

liferay-translation committed:
LPS-77699 Update Translations
